awesomeConfig.cmake

```diff
@@ -148,6 +148,8 @@ set(AWESOME_DEPENDENCIES
     xcb-icccm
     xcb-icccm>=0.3.8
     xcb-xfixes
+    xcb-composite
+    xcb-damage
```

Comment:
Those will have to be installed in CI and listed as dependencies in the docs.

Reply:
Done.